wf-core-tck-runner/cdi-tck/pom.xml

```diff
@@ -0,0 +1,511 @@
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-dependency-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>copy-dependencies</id>
+                <phase>validate</phase>
+                <goals>
+                  <goal>copy-dependencies</goal>
+                </goals>
+                <configuration>
+                  <outputDirectory>${project.build.directory}/dependency/lib</outputDirectory>
+                  <overWriteSnapshots>true</overWriteSnapshots>
+                  <includeArtifactIds>cdi-tck-ext-lib</includeArtifactIds>
+                </configuration>
+              </execution>
+            </executions>
+          </plugin>
+
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-antrun-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>update-jboss-as</id>
+                <goals>
+                  <goal>run</goal>
+                </goals>
+                <phase>validate</phase>
+                <configuration>
+                  <target>
+                    <property file="${basedir}/local.build.properties" />
+                    <property file="${basedir}/build.properties" />
+                    <property name="jboss.home" value="${env.JBOSS_HOME}" />
+                    <echo>
+                      =========================================================================
+                      Adding CDI ext library
+                      Weld version: ${weld.version}
+                      JBOSS_HOME: ${env.JBOSS_HOME}
+                      =========================================================================
+                    </echo>
+
+                    <copy todir="${env.JBOSS_HOME}/standalone/lib/ext" overwrite="true">
+                      <fileset dir="target/dependency/lib">
+                        <include name="cdi-tck-ext-lib-${cdi.tck.version}.jar" />
+                      </fileset>
+                    </copy>
+                    <copy todir="${env.JBOSS_HOME}/standalone/configuration" overwrite="true">
+                      <fileset dir="src/test/configuration">
+                        <include name="standalone-eecore.xml" />
+                      </fileset>
+                    </copy>
+
+                  </target>
+                </configuration>
+              </execution>
+            </executions>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
+
+    <profile>
+      <id>cdi-signature-test</id>
+      <!-- API jars from the WildFly distribution -->
+      <dependencies>
+        <dependency>
+          <groupId>jakarta.annotation</groupId>
+          <artifactId>jakarta.annotation-api</artifactId>
+          <scope>system</scope>
+          <systemPath>${env.JBOSS_HOME}/modules/system/layers/base/jakarta/annotation/api/main/jakarta.annotation-api-2.1.0.jar</systemPath>
```

Comment:
LATER: Use properties for all these versions.

Reply:
:+1: 

Perhaps we could set the property based on something like doing the following in the WildFly folder:
annotationapi=`find -name jakarta*annotation*api*.jar`